src/less-style/structure.less

```diff
@@ -60,29 +60,39 @@
 
 .fd-head {
```

Comment:
A bunch of these rules were getting overwritten by the second definition of `.fd-head` lower in this file